go/utils/setup.go

```diff
@@ -50,9 +85,74 @@ func SetupEnvironment() {
 
 	Network = os.Getenv("NETWORK")
 	if Network == "" {
-		Network = "testing"
+		Network = "russianhill"
```

Comment:
Here I still have an open question. I thought we running the functional tests against the nodes deployed for russianhill?

Reply:
yeah by default it uses the `network` value that is passed as env var here: https://github.com/centrifuge/functional-testing/blob/8a9d7916225b35a6046d72b3990e6e9dc0220612/kubernetes/helm/functional-testing/chart/functionalTestingChart/templates/deployment.yaml#L14 
This is just a default when running outside of kubernetes/docker, local plain `go test command`